clivendor/README.md

```diff
@@ -0,0 +1,7 @@
+# CLI Vendor hack
+
+This is a workaround for the go tooling being confused by `frameworks/myframework/cli/` pointing into `cli/` when using vendoring.
+
+Frameworks in dcos-commons/frameworks/ should create a 'vendor' symlink which points to here. In other words, a symlink at `frameworks/yoursvc/cli/dcos-yoursvc/vendor` should point to `clivendor/`.
```

Comment:
Should we add this to the `new-framework.sh` script?

Reply:
Open question: delete that script as of this PR?

Reply:
What is the documented way to add a new framework and/or create a new standalone service. But yes, the scripts that are not relevant anymore should be removed.